templates/contactDetails.html

```diff
@@ -31,11 +31,11 @@
 		</header>
 		<section>
 			<div ng-repeat="prop in ctrl.contact.props | toArray | orderDetailItems:'$key'">
-				<detailsItem ng-repeat="propData in prop" name="prop.$key" data="propData" model="ctrl" index="$index" ng-class="[ 'details-item-' + prop.$key ]"></detailsItem>
+				<detailsItem ng-repeat="propData in prop" name="prop.$key" data="propData" model="ctrl" index="$index" ng-class="[ 'details-item-' + prop.$key, {'hidden': 'org title version fn geo photo uid'.indexOf(prop.$key) !== -1} ]"></detailsItem>
```

Comment:
Because it outputs a hidden input, who breaks the display flex of the whole form.
So I need to hide them to only display the inputs we can modify.